client/question/question.js

```diff
@@ -621,10 +621,12 @@ tie.constant('RUNTIME_ERROR_FEEDBACK_MESSAGES', {
           " item assignment");
     },
     generateMessage: function() {
-      return ["Unfortunately Python doesn't support directly assigning ",
-        "characters in a string. If you need to do so, try splicing the ",
-        "string and reassigning the characters that way. If you need a ",
-        "refresher on splicing, check out the primer."].join('');
+      return [
+        "Unfortunately Python doesn't support directly assigning ",
+        "characters in a string. If you need to do so, try splitting the ",
```

Comment:
Hm. Are we talking here about something like this?

```
s = "cat"
s[0] = 'b'
assert s == "bat"
```

If so, wouldn't the most natural way to do it be something like `'b' + s[1:]`? I'm not sure I understand the "split()" approach.